dc/council/code/titles/19/sections/19-1118.xml

```diff
@@ -8,30 +8,32 @@
     <para>
       <num>(1)</num>
       <text>The execution and either delivery to the custodial trustee or recording of an instrument in substantially the following form:</text>
-      <para>
-        <heading>TRANSFER UNDER THE DISTRICT OF COLUMBIA’S</heading>
-      </para>
-      <para>
-        <heading>UNIFORM CUSTODIAL TRUST ACT</heading>
-        <text>I, ______ (name of transferor or name and representative capacity if a fiduciary), transfer to ______ (name of trustee other than transferor), as custodial trustee for ______ (name of beneficiary) as beneficiary and ______ as distributee on termination of the trust in absence of direction by the beneficiary under the District of Columbia’s Uniform Custodial Trust Act, the following: (Insert a description of the custodial trust property legally sufficient to identify and transfer each item of property).</text>
-        <text>Dated: ______</text>
-        <text>__________</text>
-        <text>(Signature); or</text>
-      </para>
+      <text>
+        <center>
+          <strong>TRANSFER UNDER THE DISTRICT OF COLUMBIA’S</strong>
+          <br/>
+          <strong>UNIFORM CUSTODIAL TRUST ACT</strong>
+        </center>
+      </text>
+      <text>I, ______ (name of transferor or name and representative capacity if a fiduciary), transfer to ______ (name of trustee other than transferor), as custodial trustee for ______ (name of beneficiary) as beneficiary and ______ as distributee on termination of the trust in absence of direction by the beneficiary under the District of Columbia’s Uniform Custodial Trust Act, the following: (Insert a description of the custodial trust property legally sufficient to identify and transfer each item of property).</text>
+      <text>Dated: ______</text>
+      <text>__________</text>
+      <text>(Signature); or</text>
```

Comment:
Just a note. In the far future, it would be helpful to parse out these ending ';or', ':', and other notable ending punctuation.